ui/imports/shared/popups/send/views/AmountToSend.qml

```diff
@@ -207,14 +225,44 @@ Control {
                                                  : root.multiplierIndex
                     locale: root.locale.name
                 }
+
+                // in case we dont have progressing font size reduction we only allow users to enter digits until there is place left.
```

Comment:
@micieslak @caybro here  I am trying to limit the number of maximum length for the TextField based on the available space. This is need as per the new design mentioned here https://www.figma.com/design/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=25233-292013&t=lyrihnGzlWTKcnI8-4 . If you can think of a better way to do it, please let me know